tinytodo/README.md

```diff
@@ -14,9 +14,11 @@ You need Python3 and Rust. Rust can be installed via [rustup](https://rustup.rs)
 
 This example expects that the [`cedar`](https://github.com/cedar-policy/cedar) repository is cloned into the toplevel (`../cedar-examples`) directory. You can instruct Cargo to use your local version of `cedar-policy` by adding `path = "../cedar/cedar-policy"` to Cargo.toml.
 
-Install the needed python packages, and build the server as follows.
+Install the needed python packages in a virtual environment, and build the server as follows.
 
 ```shell
+python3 -m venv /tmp/tinytodo
+source /tmp/tinytodo/bin/activate
```

Comment:
is it a good idea / best practice to put the virtual environment in `tmp`?  I think I've previously just put the virtual environment in the current directory, but not sure if that's good or not

Reply:
From my limited python experience I would expect the virtual env to be in the project directory

Reply:
That seems reasonable.  In fact, now that you say that, I feel like I've typed `python3 -m venv venv` before

Reply:
yeah, that's what I assumed craig meant initially